js/examples/nextjs-openai-simple/instrumentation-node.ts

```diff
@@ -0,0 +1,37 @@
+import { SimpleSpanProcessor } from "@opentelemetry/sdk-trace-base";
+import { OTLPTraceExporter } from "@opentelemetry/exporter-trace-otlp-proto";
+import { OpenAIInstrumentation } from "@arizeai/openinference-instrumentation-openai";
+import * as OpenAI from "openai";
+import { diag, DiagConsoleLogger, DiagLogLevel } from "@opentelemetry/api";
+import { registerInstrumentations } from "@opentelemetry/instrumentation";
+import { NodeTracerProvider } from "@opentelemetry/sdk-trace-node";
+import { Resource } from "@opentelemetry/resources";
+
+// For troubleshooting, set the log level to DiagLogLevel.DEBUG
+diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.INFO);
+
+const provider = new NodeTracerProvider({
+  resource: new Resource({
+    ["openinference.project.name"]: "nextjs-chat",
```

Comment:
Suggestion: Use our semantic convention constant here rather than hard-coding.

Reply:
Yeah it's not in the conventions right now. https://github.com/Arize-ai/openinference/issues/494